provider/cmd/pulumi-resource-eks/schema.json

```diff
@@ -980,6 +980,11 @@
                     "type": "string",
                     "description": "Type of Amazon Machine Image (AMI) associated with the EKS Node Group. Defaults to `AL2_x86_64`. See the AWS documentation (https://docs.aws.amazon.com/eks/latest/APIReference/API_Nodegroup.html#AmazonEKS-Type-Nodegroup-amiType) for valid AMI Types. This provider will only perform drift detection if a configuration value is provided."
                 },
+                "bootstrapExtraArgs": {
+                    "type": "string",
```

Comment:
It would, but I'm maintaining consistency with the existing `NodeGroup` resource which uses string inputs.
Ref: https://www.pulumi.com/registry/packages/eks/api-docs/nodegroup/#kubeletextraargs_nodejs